src/main/java/org/example/homework3/Task5.java

```diff
@@ -0,0 +1,17 @@
+package org.example.homework3;
+
+public class Task5 {
+    public static void main(String[] args) {
+
+        double Fahrenheit;
```

Comment:
variables names should not start from uppercase letter, here and below
```suggestion
        double fahrenheit;
```